jujugui/static/gui/src/app/app.js

```diff
@@ -483,16 +483,19 @@ YUI.add('juju-gui', function(Y) {
       // initially.
       var user = env.get('user');
       var password = env.get('password');
-      var credentials = env.getCredentials();
+      var macaroons = null;
       if (!user || !password) {
```

Comment:
So if a username and password are provided then we don't want to store macaroons? Can you add a code comment which explains the 5w of these changes plz.